lib/engine/game/base.rb

```diff
@@ -1306,6 +1306,8 @@ def place_home_token(corporation)
               home_token_locations(corporation)
             end
 
+          return unless hexes
```

Comment:
National railways may not have a home token to lay. Continue on without placing a token if they have no home hex.